### PR TITLE
navbar functionality

### PR DESCRIPTION
**Pull request by usama-ms**

### CODE REVIEW

**Review by zain-sattar — COMMENTED** [review]
Always use naming conventions that give clear meaning and context of variables and classes.

File: style.css [hunk 1]
```diff
@@ -260,9 +350,170 @@ body {
...
+
+.card-text {
+    padding-top: 10px;
+    font-size: 35px;
+    color: #2bb34d;
+    line-height: 33px !important;
+    font-weight: 550;
+}
+
+.card-one {
```
Comment (zain-sattar) [inline style.css:423]: replace it with some meaningful name

File: style.css [hunk 2]
```diff
@@ -260,9 +350,170 @@ body {
...
+
+.card-body {
+    text-align: center;
+    display: flex;
+    height: 100%;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.mosttop {
```
Comment (zain-sattar) [inline style.css:482]: here as well

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (JavedJutt) [inline .gitignore:1]: pls add eof

File: breadcrumb.html [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<nav style="--bs-breadcrumb-divider: '>';" aria-label="breadcrumb">
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item"><a href="#">HOME</a></li>
+        <li class="breadcrumb-item"><a href="#">COLLAGEN PROTEIN</a></li>
+    </ol>
+</nav>
```
Comment (JavedJutt) [inline breadcrumb.html:6]: same

File: cards.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
+<div class="package-cards d-flex">
+    <div class="one-pack">
+        <div class="card">
+            <div class="card-body">
+                <div class="text">
+                    <p class="card-title">1 Pack</p>
+                    <span class="first-card">$49</span>
+                </div>
+                <div class="first-card-btn"><a href="#"
```
Comment (JavedJutt) [inline cards.html:9]: pls format <a> tag

File: cards.html [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        <div class="card">
+            <div class="card-body">
+                <p class="card-title">6 Packs</p>
+                <span class="card-text">$31<br>Per Pack<br></span>
+                <span class="save">(Save 36%)</span>
+                <a href="#" class="btn btn-primary flavor-btn">CHOOSE<br>FLAVORS</a>
+            </div>
+        </div>
+    </div>
+</div>
```
Comment (JavedJutt) [inline cards.html:36]: eof

File: breadcrumb.html [hunk 7]
```diff
@@ -0,0 +1,6 @@
+<nav style="--bs-breadcrumb-divider: '>';" aria-label="breadcrumb">
```
Comment (JavedJutt) [inline breadcrumb.html:1]: what does  --bs- represent?

File: style.css [hunk 8]
```diff
@@ -0,0 +1,775 @@
+.navbar-bg {
+    background: #0d4f9a;
+}
+
+body {
+    top: 0;
+    background: #ffffff;
+    font-family: var(--font-stack-body, "Lato", "HelveticaNeue", "Helvetica Neue", sans-serif);
```
Comment (JavedJutt) [inline style.css:8]: format style as top on related to tags like body or p then follow them by className

File: breadcrumb.html (hunk 7 above)
Comment (usama-ms) [reply breadcrumb.html:1]: it represents custom property of bootstrap as bs stands for bootstrap

**Pull request merged**